modules/lang-painless/src/main/resources/org/elasticsearch/painless/org.elasticsearch.script.ingest.meta.txt

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0 and the Server Side Public License, v 1; you may not use this file except
+# in compliance with, at your election, the Elastic License 2.0 or the Server
+# Side Public License, v 1.
+#
+
+# This file only whitelists the script class and the Metadata class for ingest scripts.
+# Due to name conflicts, this whitelist cannot be loaded with other scripts with conflicting Metadata types.
```

Comment:
The painless test context [loads](https://github.com/elastic/elasticsearch/pull/87309/files#diff-ba2678b0d3fb572e0a19426a2dd2c323425afc15b83369a7ff08ca449c803057L111) all script whitelists.  It's not an issue in this change, but when there are multiple Metadata classes, there is a name collision.

The solution of this PR is to avoid loading meta.txt whitelists in painless_test.  Another option is to explicitly name the script whitelists needed by painless test.

Reply:
Removed